development-vm/Procfile

```diff
@@ -185,3 +185,5 @@ search-api-govuk-index-listener:  govuk_setenv search-api ./run_in.sh ../../sear
 search-api-bulk-reindex-listener: govuk_setenv search-api ./run_in.sh ../../search-api    bundle exec rake message_queue:bulk_insert_data_into_govuk
 # sidekiq-monitoring for search-api uses 3234
 content-data-api:      govuk_setenv content-data-api ./run_in.sh ../../content-data-api bundle exec rails server -p 3235
+content-publisher-sidekiq:               govuk_setenv content-publisher ./run_in.sh ../../content-publisher    bundle exec sidekiq -C ./config/sidekiq.yml
```

Comment:
Because of https://github.com/alphagov/govuk-puppet/blob/9906d082a5908dc3db3447c0a7439690c00caa0b/development-vm/Procfile#L175 running foreman start we don't actually need this line here as sidekiq will be run as part of that.

I'm not sure if there are advantages to running separate processes but if we do go down a separate process route I think we'd be better eschewing the conventions here and referencing the processes in our procfile. E.g. `foreman start worker -f Procfile.dev` as that decouples our configuration from this file.

Reply:
Ok will remove